rfc/0010-cbft.md

```diff
@@ -96,7 +96,16 @@ This RFC describes the SDK API and behavior exposure of the Couchbase Server Ful
 			- [Numeric Range Facet Results](#numeric-range-facet-results)
 			- [Date Range Facet Results](#date-range-facet-results)
 - [Language Specifics](#language-specifics)
-- [Unresolved Questions](#unresolved-questions)
+		- [Set-Up for Examples](#set-up-for-examples)
+		- [Java](#java)
+		- [.NET](#net)
+		- [NodeJS](#nodejs)
+		- [Go](#go)
+		- [C](#c)
+		- [PHP](#php)
+		- [Python](#python)
+		- [Ruby](#ruby)
+- [Resolved Questions](#resolved-questions)
```

Comment:
Maybe this section should just be called "Questions"  or I would not keep the resolution of the question in the list.